### DISCUSSION TITLE
Ideas for monetization

### DISCUSSION THREAD

**Discussion by airamez** [zloirock/core-js#1216] [General]
What about a book describing your saga with this experience with open source?
Some suggestion for te title:
- Open source pitifals
- The risks of doing the good with open source
- Open source, how to survive?

**Comment by yanagawak** [comment]
Classic case of Marginal utility.
Take an example, we cannot survive more than a few days without water. Water is more important than diamonds. Yet, water is very cheap, diamonds are expensive.
We need toilet cleaners to clean public toilets. Public toilet cleaners are more important than WWE wrestlers. Yet, toilet cleaners make very little money, and WWE wrestlers make millions.
Same logic applies to open source developers. The work they do is extremely important. But yet, they receive no payment. Why? Because of low marginal utility. There are many other projects offering similar tools. If an open source project fails, Google, Microsoft, can easily assign 5 staff and make a working fork within 6 hours.

**Reply by airamez** [reply]
Still 5 Staff is way more expense than sponsoring this guy and project.

**Reply by airamez** [reply]
I my humble opinion, big tech or startup companies could hire him, revamp the project, make it profitable and would have a brilliant software engineer with a proven record of real world success. His talent, experience, expertise and devotion could be used to different projects.
It would be a win-win situation.

**Comment by yanagawak** [comment]
My point is that there is "value" and then there is "marginal utility".
Value is what you provide, marginal utility is what you get paid.
This is a fundamental law of economics.
People have gotten around this via government regulation and emotional pleas. As a White Russian Male, he is unlikely to get any.
Underpaid open source developers should not feel guilty about leaving their project and moving on to work of higher marginal utility.
Their low pay is a signal from the market that if they abandoned their project, an equal or better alternative will quickly appear.

**Reply by airamez** [reply]
I respect you opinion but I disagree. There are many other laws of economics, it is not a uni dimensional think. I know nothing about Russian, just what we hear in the internet, movies and memes.
But I don’t think his color or gender would play a role in this.
This is not the first open source project to have similar issues and many other found a way to became profitable.
I do agree on, if it is not working and the we can pay the bill, we should try different approaches. But many visionaries took a different route and were very successfully financially.
It is never that simple and unidimensional.

**Reply by yanagawak** [reply]
When did I say this problem is "unidimensional"??

**Reply by yanagawak** [reply]
"But I don’t think his color or gender would play a role in this."
Oh yes it does.

**Reply by twisted-nematic57** [reply]
> "But I don’t think his color or gender would play a role in this." Oh yes it does.

Everybody knows it's not the fault of every Russian alive that a war is happening and people are dying. In fact there are a lot of Russians who are against it.

And I thought the world had become civilized enough by now that skin color was a property irrelevant in most situations. Now of course, there are idiots and outliers, but there's nothing we can do about that.

**Comment by zloirock** [comment]
I'm not a writer, however, thanks for the suggestion -)
